### DISCUSSION TITLE
Adding events to different calendars.

### DISCUSSION THREAD

**Discussion by rightcreative** [spatie/laravel-google-calendar#234] [Q&A]
Trying to do some digging on this. I see that there is a calendar ID specified in the .env file. Is there a way to override this, and pass the ID of a different calendar, when creating an event?

**Comment by tomcodes** [comment]
Just rely on Laravel config management for this, see here: https://github.com/spatie/laravel-google-calendar/discussions/179#discussioncomment-3373280
